angr/sim_type.py

```diff
@@ -127,7 +127,10 @@ def __repr__(self):
         return "reg{}_t".format(self.size)
 
     def extract(self, state, addr, concrete=False):
-        out = state.memory.load(addr, self.size / 8, endness=state.arch.memory_endness)
+        # TODO: EDG says this looks dangerously closed-minded. Just in case...
+        assert self.size % state.arch.byte_width == 0
+
+        out = state.memory.load(addr, self.size / state.arch.byte_width, endness=state.arch.memory_endness)
```

Comment:
can we instead implement that self.size is in bytes instead of bits? I recommended that earlier, did that not actually happen?

Reply:
this and the below divisions can be removed bc of that so I'm not going to comment "floor division" on every single one of them

Reply:
In the interest of my sanity, I think I will propose to punt that to feature/multiverse_exploration or something
But I fixed the division, of course (commit incoming)